src/components/ConfigViewer/columns.js

```diff
@@ -61,7 +61,7 @@ export function DateCell({ row, column }) {
   const dateString = row?.values[column.id];
   return (
     <div className="text-xs">
-      {dateString ? dayjs(dateString).format("YYYY-DD-MM h:mm A") : "None"}
+      {dateString ? timeago.format(dateString) : "None"}
```

Comment:
nit: If there is a chance where this conversion fails with exception, we need to wrap this conversion using try catch.

Reply:
Ack. Just converted the formatting. For now, I will let it be. We can check the logic if it becomes an issue.